Readme.md

```diff
@@ -13,34 +13,48 @@ This project allows you to use predefined build scripts for *SCSS* files without
 
-Then, add a project reference to _Lombiq.NodeJs.Extensions.csproj_ and add the following `Import` statements to your project file:
+Then, add a project reference to _Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.csproj_ and the following `Import` statements to your project file:
 
 ```xml
-<Import Project="..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.props"/>
-<Import Project="..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.targets"/>
+<!-- At the top: -->
+<Import Project="..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.props" />
+
+<!-- At the bottom: -->
+<Import Project="..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.targets" />
 ```
+
 In case you've placed the submodule in a different location, adjust the paths as necessary.
 
 ### As a NuGet package
 
-When adding _Lombiq.NodeJs.Extensions_ as a NuGet package, no further steps are necessary.
+When adding `Lombiq.NodeJs.Extensions` as a NuGet package, no further steps are necessary.
 
 ### Integration with MSBuild
 
-_Lombiq.NodeJs.Extensions_ tightly integrates with the MSBuild build system and executes linting and compilation tasks transparently. In case of warnings or errors during the execution of those tasks, respective warnings and errors will be generated with MSBuild and brought to your attention.
+`Lombiq.NodeJs.Extensions` tightly integrates with MSBuild and executes linting and compilation tasks transparently. In case of warnings or errors during the execution of those tasks, respective MSBuild warnings and errors will be generated and surfaced.
+
+During the first build of your project after adding `Lombiq.NodeJs.Extensions`, it will additionally be added as an `npm` package to your project, which allows you to run the contained `npm` scripts from your project. Refer to the [available scripts](#available-scripts) section for more information.
+
+### Troubleshooting
 
-During the first build of your project after adding _Lombiq.NodeJs.Extensions_, it will additionally be added as an `npm` package to your project, which allows you to run the contained `npm` scripts from your project. Refer to the [available scripts](#available-scripts) section for more information.
+Should you encounter the following error:
+
+```
+ΓÇëERRORΓÇë ENOENT: no such file or directory, realpath [...]
+```
+... then please try and override the `NodeJsExtensionsNpmPackageSourcePath` property with something shorter than the default value of `./node_modules/.nx`. You can try to use `.nx`, but you then need to add *.nx* to your *.gitignore* file. Should that still not work, please use the relative path to your solution root or something similar.
```

Comment:
```suggestion
...then please try and override the `NodeJsExtensionsNpmPackageSourcePath` property with something shorter than the default value of `./node_modules/.nx`. You can try to use `.nx`, but you then need to add *.nx* to your *.gitignore* file. Should that still not work, please use the relative path to your solution root or something similar.
```

Reply:
Can you point to a source for this suggestion?

Reply:
Weird that you see this already because I didn't finish and send my review yet.

There seems to be no definite spelling rule on spaces before/after (or even within) an ellipsis, this being up to the given style. However, some indications point to it not being valid as the start of a sentence anyway, so I'll add a different suggestion.